tests/target/structs.rs

```diff
@@ -356,3 +356,31 @@ where
 struct Parens((i32, i32))
 where
     i32: Trait;
+
+// #5215
+struct MyTuple(
+    /// Doc Comments
+    // TODO note to add more to Doc Comments
+    u32,
+    /// Doc Comments
+    // TODO note
+    u64,
+);
+
+// #5215
+struct MyTuple(
+    #[cfg(unix)] // some comment
+    u64,
+    #[cfg(not(unix))] // block comment
+    u32,
+);
+
+// #5215
+struct MyTuple(
+    #[cfg(unix)]
+    // some comment
+    u64,
+    #[cfg(not(unix))]
+    // block comment
+    u32,
+);
```

Comment:
Thanks again for adding these additional tests! On closer inspection of the tests it strikes me as odd that the block comments are changed to inline comments. I understand this is happening becuase this file sets``// rustfmt-normalize_comments: true``.

I think it would be nicer to isolate these tests. Would it be possible to move these tests to ``tests/target/issue_5215.rs`` and ``tests/source/issue_5215.rs``. Feel free to use a different name for the file, just want to make sure we highlight the link between these tests and the corresponding issue.